src/RestQuery.js

```diff
@@ -846,6 +871,26 @@ function includePath(config, auth, response, path, restOptions = {}) {
     }
   }
 
+  if (restOptions.excludeKeys) {
+    const keys = new Set(restOptions.excludeKeys.split(','));
```

Comment:
Can you refactor this since it's the same as keys?

Reply:
I did the same thing here, so you want this to say `excludeKeys`?